assets/js/cookieBanner.js

```diff
@@ -0,0 +1,51 @@
+const cookieBanner = document.getElementById('cookie-banner')
```

Comment:
Maybe move just to the block where you use this and pass it into `hideCookiesConfirmation`? Everything in here is in global scope too, might be worth dropping the block of code into a self-executing (closure?) block to keep it tidy and self-contained?

Reply:
Yeah, good point - I'll wrap it up!

Reply:
I've updated this to pop it in its own block so hopefully better now 👍